### PR TITLE
CORTX-32211: PEP 257 tool Codacy issues

### PR DESCRIPTION
**Pull request by hessio**
Solution: Follow docstring conventions as outlined in
tool: https://peps.python.org/pep-0257/

Signed-off-by: hessio <patrick.hession@seagate.com>

### CODE REVIEW

File: hax/hax/motr/planner.py [hunk 1]
```diff
@@ -274,7 +282,9 @@ def _inc_group(self):
             state.next_group_commands = set()
 
     def notify_finished(self, command: BaseMessage) -> None:
-        ''' The method must be invoked by the worker thread when the command
+        ''' mthod invoked by the worker thread when the command is executed.
```
Comment (d-nayak) [inline hax/hax/motr/planner.py:285]: "Method" ... ?

**Review by d-nayak — CHANGES_REQUESTED** [review]

**Review by vaibhavparatwar — APPROVED** [review]

**Comment by Shreya-18** [comment]
@hessio I can still new 14 codacy issues added

https://app.codacy.com/gh/Seagate/cortx-hare/pullRequest?prid=9634694&bid=29705438

Are we not supposed to address them?

**Comment by vaibhavparatwar** [comment]
retest this please

File: hax/hax/consul/cache.py [hunk 2]
```diff
@@ -67,6 +67,8 @@ def __repr__(self):
 
 def supports_consul_cache(f: T) -> T:
     """
+    create the instance of cache that nested functions can pick up.
```
Comment (Shreya-18) [inline hax/hax/consul/cache.py:70]: create -> Create

File: hax/hax/motr/planner.py [hunk 3]
```diff
@@ -230,7 +232,10 @@ def _is_allowed_asap(self, command: BaseMessage) -> bool:
         return True
 
     def _is_allowed(self, command: BaseMessage) -> bool:
-        ''' The command is allowed for execution if and only if the command has
+        '''
+        returns True group_id equal to the currently active group
```
Comment (Shreya-18) [inline hax/hax/motr/planner.py:236]: returns -> Returns

File: hax/hax/motr/planner.py [hunk 4]
```diff
@@ -274,8 +283,10 @@ def _inc_group(self):
             state.next_group_commands = set()
 
     def notify_finished(self, command: BaseMessage) -> None:
-        ''' The method must be invoked by the worker thread when the command
-            is executed.
+        '''method invoked by the worker thread when the command is executed.
```
Comment (Shreya-18) [inline hax/hax/motr/planner.py:286]: method -> Method

File: provisioning/miniprov/hare_mp/cdf.py [hunk 5]
```diff
@@ -483,6 +483,8 @@ def _get_m0d_per_cvg(self, machine_id: str, cvg: int) -> int:
 
     def _get_node_clients(self, machine_id: str) -> Iterator[M0ClientDesc]:
         """
+        get motr client nodes.
```
Comment (Shreya-18) [inline provisioning/miniprov/hare_mp/cdf.py:486]: get-> Get

**Review by Shreya-18 — CHANGES_REQUESTED** [review]
Please look at the minor changes. Also squash all the commits.

**Comment by Shreya-18** [comment]
@hessio Looks like 7 new codacy issues are added. :(
https://app.codacy.com/gh/Seagate/cortx-hare/pullRequest?prid=9634694&bid=29705438

Please have a look!

**Review by mssawant — APPROVED** [review]

**Review by Shreya-18 — APPROVED** [review]

**Review by Shreya-18 — APPROVED** [review]

**Review by d-nayak — APPROVED** [review]

**Pull request merged**